src/NHibernate/Collection/Generic/SetHelpers/SetSnapShot.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-#if NETCOREAPP2_0_OR_GREATER || NETSTANDARD2_1_OR_GREATER
+#if NETCOREAPP2_0_OR_GREATER || NETSTANDARD2_1_OR_GREATER || NET5_0_OR_GREATER
```

Comment:
Am I wrong thinking `NETCOREAPP2_0_OR_GREATER` already implies `NET5_0_OR_GREATER`, causing this change and most other changes to be unneeded?